package.json

```diff
@@ -36,6 +36,12 @@
       "hash": "QmZvXudUXTc2G3o2SYPU7uQX1gzbeQkAZXJ6bnN8C4rcXE",
       "name": "go-libp2p-rpc",
       "version": "0.0.10"
+    },
+    {
+      "author": "hsanjuan",
+      "hash": "QmSNNKmo6LtoiSLjaVJxi76QFRfKJvJUME96EmtLEtFPdq",
+      "name": "cli",
+      "version": "2017.1.24"
```

Comment:
Yes indeed that's better. These were actually my excuses for other projects which do not provide release tags and forgot to check this time. I'll fix it.